model/variable.cc

```diff
@@ -415,6 +415,20 @@ bool VariableBase::visible() const
   auto g=group.lock();
   //toplevel i/o items always visible
   if ((!g || !g->group.lock()) && g==controller.lock()) return true;
+  // ensure pars and constants with invisible out wires are made invisible. for ticket 1275  
+  if (type()==constant || type()==parameter)
+  {
+    for (auto w: ports[0]->wires())
+      if (w->attachedToDefiningVar()) 
+      {
+        if (w->visible()) return true;
```

Comment:
Can be simplified to "return w->visible()"

Reply:
But I think the code is still wrong. This will return false when the first outgoing wire is invisible, even if the remaning wires are visible = dangling wires.

What you should do is set a boolean flag that tracks whether any wire is attachedToDefiningVar. Then return false on line 428 if that flag is set. Also I think the code on line 429 is now obsolete.

Reply:
Thanks for the tips :)

> What you should do is set a boolean flag that tracks whether any wire is attachedToDefiningVar.

I tried using std::any_of, but it didn't work previously. Now it seemingly does with this:

```
if (std::any_of(ports[0]->wires().begin(),ports[0]->wires().end(), [](Wire* w){return w->attachedToDefiningVar() && !w->visible();})) return false;
	else return true;
```

> Also I suspect the code on line 429 is now obsolete.

I thought so too, but when I remove it vars attached to plots still disappear in the fundamentalConstants.mky example.